pkg/operator/encryption/controllers/state_controller.go

```diff
@@ -103,6 +105,78 @@ func (c *stateController) sync() error {
 	return configError
 }
 
+type eventWithReason struct {
+	reason  string
+	message string
+}
+
+// eventsFromEncryptionConfigChanges return slice of event reasons with messages corresponding to a difference between current and desired encryption state.
+func eventsFromEncryptionConfigChanges(current, desired map[schema.GroupResource]state.GroupResourceState) []eventWithReason {
```

Comment:
nit: could you move it down to the bottom? It would be the last function in the file that doesn't belong to  `stateController` controller.

Reply:
if you decide to do so then `eventWithReason` as well.

Reply:
sure